src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -839,59 +709,62 @@ private void DrawPackages(IEnumerable<INugetPackage> packages, bool canBeSelecte
         /// </summary>
         private void DrawOnlineHeader()
         {
-            var headerStyle = GetHeaderStyle();
-
-            EditorGUILayout.BeginVertical(headerStyle);
+            EditorGUILayout.BeginVertical(Styles.BackgroundStyle);
             {
-                EditorGUILayout.BeginHorizontal();
+                EditorGUILayout.BeginHorizontal(EditorStyles.toolbar);
                 {
-                    var showPrereleaseTemp = EditorGUILayout.Toggle("Show Prerelease", showOnlinePrerelease);
+                    var showPrereleaseTemp = GUILayout.Toggle(
+                        showOnlinePrerelease,
+                        showPrereleaseContent,
+                        EditorStyles.toolbarButton,
+                        GUILayout.Width(120f));
                     if (showPrereleaseTemp != showOnlinePrerelease)
                     {
                         showOnlinePrerelease = showPrereleaseTemp;
                         UpdateOnlinePackages();
                     }
 
+                    GUILayout.FlexibleSpace();
                     DrawSelectFromClipboardButton();
                     DrawMandatoryButtons();
                 }
 
                 EditorGUILayout.EndHorizontal();
 
-                var enterPressed = Event.current.Equals(Event.KeyboardEvent("return"));
-
-                EditorGUILayout.BeginHorizontal();
+                var style = new GUIStyle(EditorStyles.toolbar) { fixedHeight = 25f };
```

Comment:
This style is used in multiple places. Why not cache it in a field and reuse it instead of creating it all the time?

Reply:
I cached it inside the Styles class.